source/manual/covid-19-services.html.md

```diff
@@ -260,6 +260,21 @@ following dashboards:
 GOV.UK Production Corona Forms *TODO: where?*
 GOV.UK Staging Corona Forms *TODO: where?*
 
+## Extracting form responses (business only)
+
+Cabinet Office will require data exports at regular intervals.
```

Comment:
Can we make this explicitly reference the "business volunteer form"? There are MUCH more stringent data controls around who has access to the vulnerable people data - I don't know what those are, but we shouldn't document them - if you know you know, you know?

Reply:
I'll track down some answers for this one.

Reply:
looks like that is amended in the title?